trio/testing/_fake_net.py

```diff
@@ -42,7 +42,7 @@ def _wildcard_ip_for(family: int) -> IPAddress:
     elif family == trio.socket.AF_INET6:
         return ipaddress.ip_address("::")
     else:
-        assert False
+        raise AssertionError()
```

Comment:
here

Reply:
Changed in a5afed7 and added reason in e1fdebc8